driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudHelper.java

```diff
@@ -425,7 +425,11 @@ OperationResult executeDeleteMany(final BsonDocument operation) {
     }
 
     private BsonDocument toExpected(final DeleteResult result) {
-        return new BsonDocument("deletedCount", new BsonInt32((int) result.getDeletedCount()));
+        if (result.wasAcknowledged()) {
```

Comment:
Some of the new tests include unacknowledged writes, which must be handled specially.